lib/plugins/inspec-suggest/lib/inspec-suggest/suggtool/bump.rb

```diff
@@ -0,0 +1,66 @@
+require_relative "base"
+
+module InspecPlugins::Suggest
+  class SuggToolCommand < Inspec.plugin(2, :cli_command)
+
+    option :patch, type: :boolean, default: true,
+      desc: "Increase third digit of the version number"
+    option :minor, type: :boolean, default: false,
+      desc: "Increase second digit of the version number"
+    option :major, type: :boolean, default: false,
+      desc: "Increase first digit of the version number"
```

Comment:
Having three different options creates the possibility of running with incoherent options - like `inspec suggtool bump --major --minor`.  I would suggest having one option that takes a parameter, like `--level [patch|minor|major]` default patch.

Reply:
Right, changed accordingly.